docker-storage-setup.spec

```diff
@@ -54,9 +38,12 @@ install -p -m 755 %{SOURCE5} %{buildroot}/%{dsslibdir}/dss-child-read-write
 %files
 %{_unitdir}/docker-storage-setup.service
 %{_bindir}/docker-storage-setup
-%{dsslibdir}/docker-storage-setup
+%config(noreplace) %{_sysconfdir}/sysconfig/docker-storage-setup-override
```

Comment:
I thought %files section is supposed to keep a list of files which are installed by package on system. And `docker-storage-setup-override` is not installed by package on the host.